src/mongo_logger.ts

```diff
@@ -430,10 +430,30 @@ export class MongoLogger {
   maxDocumentLength: number;
   logDestination: MongoDBLogWritable | Writable;
 
+  /**
+   * This method should be used when logging errors that do not have a public driver API for
+   * reporting errors.
+   **/
```

Comment:
The doc comments here have a bit of formatting differences than the rest of the code, I don't think we have linting for this: `**/` -> `*/`

Ex.
```js
  /**
   * Close the client and its underlying connections
   *
   * @param force - Force close, emitting no events
   */
```